scripts/preBuildHTML.py

```diff
@@ -1,4 +1,17 @@
-import os
+from subprocess import call
+import os.path
+
 
 def preBuildHTMLFun():
-    os.system("npm run build")
\ No newline at end of file
+    webpackInst = os.path.isdir("../node_modules/webpack")
+
+    if webpackInst == False:
+        print("Running npm install...")
+        call(["npm", "install"])
+        print("Running npm audit fix...")
+        call(["npm", "audit", "fix"])

```

Comment:
The call to `npm audit fix` can be removed, because this defeats the purpose of `npm ci`. It will be my responsibility to keep the package.json up to date and issue-free (which I should probably take a look at).